src/FeatBit.OpenFeature.ServerProvider/ValueComparer.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Immutable;
+using OpenFeature.Model;
+
+namespace FeatBit.OpenFeature.ServerProvider;
+
+// This code is generated by GitHub Copilot
+internal class ValueComparer : IEqualityComparer<Value>
```

Comment:
As far as I can tell, this `ValueComparer` is only used by the tests, so should really be in the test project.

Reply:
done in fc5fe17